build.gradle

```diff
@@ -216,7 +216,14 @@ processResources {
     from(sourceSets.main.resources.srcDirs) {
         include 'META-INF/mods.toml'
 
-        expand 'version': artifact_version, 'MCversion': minecraft_version, 'forge_version': forge_version_min
+        expand 'version': artifact_version,
+            'minecraft_version_min': minecraft_version_min,
```

Comment:
We could put two of these per line (min and max). I just don't want a giant line of 8 variables